alchemiscale/interface/client.py

```diff
@@ -243,6 +257,25 @@ def set_network_weight(self, network: ScopedKey, weight: float) -> None:
         """
         self._post_resource(f"/networks/{network}/weight", weight)
 
+    async def _set_network_weight(self, networks: List[ScopedKey], weight) -> None:
+        data = dict(networks=[str(network) for network in networks], weight=weight)
+        await self._post_resource_async("/bulk/networks/weight/set", data=data)
+
+    def set_networks_weight(
+        self,
+        networks: List[ScopedKey],
+        weight: float,
+        batch_size: int = 1000,
+    ) -> None:
```

Comment:
Actually, want this to return the `ScopedKey`s of `AlchemicalNetwork`s that were updated, in the same order as given in `networks`. If a given `AlchemicalNetwork` doesn't exist, ``None`` returned in its place.

This is consistent with how e.g. `set_tasks_status` works.

Reply:
The original `get_network_weight` returns a `None`. Do you want me to change that, too?

Reply:
I suppose so, go for it!